crates/sui-sdk/src/lib.rs

```diff
@@ -197,25 +400,30 @@ impl Debug for RpcClient {
     }
 }
 
+/// ServerInfo contains all the useful information regarding the API version, the available RPC calls, and subscriptions.
 struct ServerInfo {
     rpc_methods: Vec<String>,
     subscriptions: Vec<String>,
     version: String,
 }
 
 impl SuiClient {
+    /// Returns a list of available RPC calls.
     pub fn available_rpc_methods(&self) -> &Vec<String> {
         &self.api.info.rpc_methods
     }
 
+    /// Returns a list of available subscriptions.
     pub fn available_subscriptions(&self) -> &Vec<String> {
         &self.api.info.subscriptions
     }
 
+    /// Returns the API version information.
```

Comment:
More detail needed here, given the return value is so loosely typed -- what's the format of the version, what is the version tied to, how is it calculated? I.e. I think this is the Cargo version of the crate that the RPC was built at.

Reply:
Not sure how to find this. Top `Cargo.toml` states that several crates get the version defined in the top-level toml file, but json-rpc is not one of them.

Reply:
The RPC version is not actually defined by the `sui-json-rpc` crate, confusingly!

Reply:
Figured it out - it is retrieved from the RPC discover service method, which uses the RPC spec. Updated the comment accordingly.